configure: error: freetype-config not found.

Hi, when building the base image I got this error

```
#7 4.429 If configure fails try --with-xpm-dir=<DIR>
#7 4.430 configure: error: freetype-config not found.
------
executor failed running [/bin/sh -c docker-php-ext-configure gd     --enable-gd-native-ttf     --with-freetype-dir=/usr/include/freetype2     --with-jpeg-dir=/usr/include/]: exit code: 1
```

Quick fix

Replace:
```
FROM php:7.2-fpm
```

to:
```
FROM php:7.2-fpm-stretch
```

--- a/base-php-nginx.Dockerfile
+++ b/base-php-nginx.Dockerfile
@@ -1,4 +1,4 @@
-FROM php:7.2-fpm
+FROM php:7.2-fpm-stretch
 
 
 ENV COMPOSER_VERSION=1.6.5 \
@@ -98,19 +98,11 @@ ENV PATH="/composer/vendor/bin:$PATH" \
     COMPOSER_ALLOW_SUPERUSER=1 \
     COMPOSER_VENDOR_DIR=/var/www/vendor \
     COMPOSER_HOME=/composer
-RUN curl -s -f -L -o /tmp/installer.php https://raw.githubusercontent.com/composer/getcomposer.org/da290238de6d63faace0343efbdd5aa9354332c5/web/installer \
- && php -r " \
-    \$signature = '669656bab3166a7aff8a7506b8cb2d1c292f042046c5a994c43155c0be6190fa0355160742ab2e1c88d40d5be660b410'; \
-    \$hash = hash('SHA384', file_get_contents('/tmp/installer.php')); \
-    if (!hash_equals(\$signature, \$hash)) { \
-        unlink('/tmp/installer.php'); \
-        echo 'Integrity check failed, installer is either corrupt or worse.' . PHP_EOL; \
-        exit(1); \
-    }" \
- && php /tmp/installer.php --no-ansi --install-dir=/usr/bin --filename=composer --version=${COMPOSER_VERSION} \
- && rm /tmp/installer.php \
- && composer --ansi --version --no-interaction
 
+COPY ./install-composer.sh /tmp//install-composer.sh
+
+RUN /tmp/install-composer.sh && rm -f /tmp//install-composer.sh \
+ && composer --ansi --version --no-interaction
 
 # install node for running gulp at container entrypoint startup in dev
 # copied from official node Dockerfile

--- a/install-composer.sh
+++ b/install-composer.sh
@@ -1,0 +1,16 @@
+#!/bin/sh
+
+EXPECTED_SIGNATURE="$(curl https://composer.github.io/installer.sig)" 
+php -r "copy('https://getcomposer.org/installer', '/tmp/composer-setup.php');" 
+ACTUAL_SIGNATURE="$(php -r "echo hash_file('SHA384', '/tmp/composer-setup.php');")" 
+  
+if [ "$EXPECTED_SIGNATURE" != "$ACTUAL_SIGNATURE" ] 
+then 
+    >&2 echo 'ERROR: Invalid installer signature' 
+    rm /tmp/composer-setup.php 
+    exit 1 
+fi 
+
+php /tmp/composer-setup.php --no-ansi --install-dir=/usr/bin --filename=composer --version=${COMPOSER_VERSION} \
+ && rm /tmp/composer-setup.php \
+ && composer --ansi --version --no-interaction

